wp-content/themes/core/pcss/base/grid/_columns.pcss

```diff
@@ -6,7 +6,6 @@
 
 .g-col {
 	min-width: 0;
-	position: relative;
```

Comment:
Any particular reason for removing this? 😃

Reply:
It would make child elements that are fixed or absolute to snap to the boundaries of g-col, because of our "categories" layout for related posts, they go all the way to the left in desktop, exceeding, but remain in position in mobile. Personally, I don't like position: relative anywhere unless we need it, against putting it somewhere as default and then do position: static. But was we don't have styles for related posts anymore, I'll revert this one.

Reply:
That makes sense and I don't disagree. @nicholasmelwell do you happen to remember or see any issue with nixing this for this base grid helper class?

Reply:
Yep, no reason not to remove it. If it's necessary it can be added on a project-by-project basis.